.github/workflows/docker-image.yml

```
@@ -0,0 +1,36 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'libp2p-server-**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v3
```

Slight preference for keeping these as short as possible.